integration-tests/data/allTestsCases.ts

```diff
@@ -1367,3 +1367,70 @@ export const commonCases: TestCase[] = [
     },
   },
 ];
```

Comment:
the whitespacing in this file is kind of inconsistent, could you do alt+shift+F on it